assembly/package-res/biserver/pentaho-solutions/system/dialects/sqlserver/applicationContext-spring-security-hibernate.properties

```diff
@@ -0,0 +1,5 @@
+jdbc.driver=com.microsoft.sqlserver.jdbc.SQLServerDriver
+jdbc.url=jdbc:sqlserver://localhost:1433;DatabaseName=hibernate
```

Comment:
@aliakseihaidukou  we are making this change for PDI  only. This change will effect ba server. So we do not want to this right now. Lets keep this change to pdi only